src/combobox/combobox.component.ts

```diff
@@ -86,6 +91,7 @@ import { filter } from "rxjs/operators";
 						ariaLabel="Close menu">
 					</ibm-icon-chevron-down16>
 			</div>
+			<div *ngIf="invalid" class="bx--form-requirement">A valid value is required</div>
```

Comment:
invalidText input can be integrated to as in dropdowns or inputs to customize this invalid message

Reply:
💯 agreed, there should be an `invalidText` input like in `dropdown.component`